cmake/AddInputFileTests.cmake

```diff
@@ -3,6 +3,10 @@
 
 spectre_define_test_timeout_factor_option(INPUT_FILE "input file")
 
+option(SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY "Minimum priority of input file \
+tests to run. Possible values are: low (not usually run on CI), normal \
+(run at least once on CI), high (run always on CI)." "normal")
+
```

Comment:
This looks great! A minor change I made to get this to work:
```diff
diff --git a/cmake/AddInputFileTests.cmake b/cmake/AddInputFileTests.cmake
index fa83ad7c17..fda13edee4 100644
--- a/cmake/AddInputFileTests.cmake
+++ b/cmake/AddInputFileTests.cmake
@@ -7,6 +7,26 @@ option(SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY "Minimum priority of input file \
 tests to run. Possible values are: low (not usually run on CI), normal \
 (run at least once on CI), high (run always on CI)." "normal")

+# Convert priority string to a number.
+if (NOT SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY)
+  set(SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY "normal")
+endif()
+string(TOLOWER "${SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY}"
+  SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY)
+message("${SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY}")
+if (${SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY} STREQUAL "high")
+  set(SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY 1)
+elseif (${SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY} STREQUAL "normal")
+  set(SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY 0)
+elseif (${SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY} STREQUAL "low")
+  set(SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY -1)
+else()
+  message(FATAL_ERROR "Unknown priority in option "
+    "SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY: "
+    "${SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY}. "
+    "Possible values are: high, normal, low")
+endif()
+
 # Environment variables for test
 set(_TEST_ENV_VARS "")
 # - Disable ASAN's leak sanitizer because Charm++ has false positives
@@ -121,23 +141,7 @@ function(add_input_file_tests INPUT_FILE_DIR)
       message(FATAL_ERROR "Unknown priority in input file ${INPUT_FILE}: "
         "${INPUT_FILE_PRIORITY}. Possible values are: high, normal, low")
     endif()
-    if (NOT ${SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY})
-      set(SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY "normal")
-    endif()
-    string(TOLOWER "${SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY}"
-      SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY)
-    if (${SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY} STREQUAL "high")
-      set(SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY 1)
-    elseif (${SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY} STREQUAL "normal")
-      set(SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY 0)
-    elseif (${SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY} STREQUAL "low")
-      set(SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY -1)
-    else()
-      message(FATAL_ERROR "Unknown priority in option "
-        "SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY: "
-        "${SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY}. "
-        "Possible values are: high, normal, low")
-    endif()
+
     # Only add tests with at least the priority specified in
     # `SPECTRE_INPUT_FILE_TEST_MIN_PRIORITY`
     if (${INPUT_FILE_PRIORITY} LESS
```

Reply:
Thanks! I squashed this in.

Reply:
Thanks!